script/Arcueid/Day 7/07_00_ARC07_3_1_3.txt

```diff
@@ -250,6 +253,7 @@ Greater Dead Apostles do not necessarily show absolute obedience toward their pa
 [sha:45d60f010415a4ccb7d1416c12523adaf00dae74]{
 -- Page 593, Offset 18059.
 -- 「上級死徒は機会さえあれば親基である祖を倒して、その呪いを受け継ぐ事ができる。
+//Not only can they oppose them, they can even overthrow them
 "If given the opportunity, Greater Dead Apostles can overthrow their parent Ancestor and inherit their curse.
```

Comment:
Not only that; if given the opportunity, Greater Dead Apostles can overthrow their parent Ancestor and inherit their curse.